components/cam/src/physics/cam/micro_p3.F90

```diff
@@ -3375,6 +3375,12 @@ subroutine cloud_sedimentation(kts,kte,ktop,kbot,kdir,   &
 
             call generalized_sedimentation(kts, kte, kdir, k_qxtop, k_qxbot, kbot, Co_max, dt_left, prt_accum, inv_dzq, inv_rho, rho, num_arrays, vs, fluxes, qnr)
 
+            !Update _incld values with end-of-step cell-ave values
+            !Note that lcldm is set in interface to have min of mincld=1e-4
```

Comment:
@PeterCaldwell if lcldm is mincld, then qc_incld is much larger than qc, and the same for nc_incld. Just a check so we're all sure it's a deliberate choice: Is it really intended to use these _incld quantities in the table lookup for fall speed?

Reply:
Yes, incld rather than cell-ave values should be used to compute fall speeds because we want to capture big drops falling fast in intense but small rain shafts instead of smearing mass and number over the entire grid cell, which would look like small drops falling slowly. Kobby asked about this too, so I should add a better comment.